rules/javascript/shared/third_parties/sanitize_html_sanitizer.yml

```diff
@@ -2,6 +2,8 @@ type: shared
 languages:
   - javascript
 patterns:
+  - $<_>.sanitize($<!>$<_>$<...>)
+  - sanitize($<!>$<_>$<...>)
```

Comment:
This shared rule was only the `sanitize-html` library before. Can we add comments for which lib these come from please? The description needs changing too.